README.md

```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
 
-## Available Scripts
+> Math Magicians
```

Comment:
- [ ] Kindly edit the project name and project description to be like the Following:
```
# Math Magicians

> (Write your project description here ...)
```